src/domains/wallet-app/api/payment-wallet/v1/_post_wallets_with_pm_policy.xml.tpl

```diff
@@ -0,0 +1,39 @@
+<policies>
+    <inbound>
+      <set-variable  name="walletToken"  value="@(context.Request.Headers.GetValueOrDefault("Authorization", "").Replace("Bearer ",""))"  />
+      <!-- Session PM START-->
+      <send-request ignore-error="true" timeout="10" response-variable-name="pm-session-body" mode="new">
+          <set-url>@($"{{pm-host}}/pp-restapi-CD/v1/users/actions/start-session?token={(string)context.Variables["walletToken"]}")</set-url>
+          <set-method>GET</set-method>
+      </send-request>
+       <choose>
+        <when condition="@(((IResponse)context.Variables["pm-session-body"]).StatusCode != 200)">
+          <return-response>
+            <set-status code="502" reason="Bad Gateway" />
+          </return-response>
+        </when>
+      </choose>
+      <set-variable name="pmSession" value="@(((IResponse)context.Variables["pm-session-body"]).Body.As<JObject>())" />
+      <return-response>
+        <set-status code="201" />
+        <set-header name="Content-Type" exists-action="override">
+          <value>application/json</value>
+        </set-header>
+        <set-body>@{
+            return new JObject(
+                          new JProperty("redirectUrl", $"https://dev.payment-wallet.pagopa.it/onboarding#sessionToken={((string)((JObject) context.Variables["pmSession"])["data"]["sessionToken"])}")
```

Comment:
Maybe the redirect url should be expressed as a variable to be interpolated for each environment?

Reply:
yep :-)